templates/docs/examples/utilities/text-max-width.html

```diff
@@ -0,0 +1,12 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Text max width{% endblock %}
+
+{% block content %}
+
+<ul class="p-list u-text-max-width">
+  <li class="p-list__item">List item 1</li>
+  <li class="p-list__item">List item 2</li>
+  <li class="p-list__item">This is a sentence long enough to show the <code>u-text-max-width</code> class adding a max width to text within a list element</li>
+</ul>
+
+{% endblock content %}
```

Comment:
We'll also need this included in a section under "Utilities" in the side navigation of the docs. To make it obvious, I would also add a paragraph to this example and a before/after comparison.
The text could say something like this: 

"
By default, unordered/ordered lists do not have a max-width, as they may be used for various purpouses (e.g. navigation). When they contain text only, we recommend adding the `.u-text-max-width` utility to the `ul/ol` element to ensure the list's width matches that of a paragraph:

This is an example paragraph to show where paragraphs wrap because of the max-width applied to them.
{List example here without the class}
{List example here with the class}
" 
Or something along those lines.

Reply:
Not at all! Thanks for the help. Just still getting used to everything I need to add. Added it to the docs now, let me know if I've missed anything else